rosdep/python.yaml

```diff
@@ -3483,6 +3483,10 @@ python-pyxhook-pip:
   ubuntu:
     pip:
       packages: [pyxhook]
+python-pyzbar:
```

Comment:
Since this is a pip-only package, please add a `-pip` suffix.  Also, while we are at it, we may as well add entries for debian and fedora as well.  Thanks.

Reply:
Thanks for the feedback. I've fixed it now. With the new suffix i should use the full name `python-pyzbar-pip` in my `package.xml` dependencies, right?

Reply:
> With the new suffix i should use the full name `python-pyzbar-pip` in my `package.xml` dependencies, right?

Correct.